hardware_interface/include/hardware_interface/internal/interface_manager.h

```diff
@@ -199,18 +225,20 @@ class InterfaceManager
         // save the combined interface for if this is called again
         interfaces_combo_[type_name] = iface_combo;
         num_ifaces_registered_[type_name] = iface_list.size();
-      } else {
-        // it is not a ResourceManager
-        ROS_ERROR("You cannot register multiple interfaces of the same type which are "
-                  "not of type ResourceManager. There is no established protocol "
-                  "for combining them.");
-        iface_combo = nullptr;
```

Comment:
Rather than deleting this altogether, can you leave the explicit (redundant) `iface_combo = nullptr` and change the `ROS_ERROR` to a comment saying something along the lines of "Multiple interfaces of the same type which are not ResourceManager cannot be combined, so return nullptr".

I think that that makes it clearer how this condition is handled.

Reply:
Sure, that seems reasonable (i don't really have to make navigating these files any harder...). Applied in rebased commits.